pkg/hostagent/setup.go

```diff
@@ -284,6 +302,19 @@ func (agent *HostAgent) configureContainerIfaces(metadata *md.ContainerMetadata)
 			}
 		}
 
+		for _, ip := range iface.IPs {
+			if iface.HostVethName != "" && iface.Mac != "" {
```

Comment:
why not just have an else case on the err != nil check that breaks out of the loop?